sdk/search/search-documents/README.md

```diff
@@ -62,19 +62,23 @@ Once you have an Admin Key, you can use it as follows:
 ```js
 const {
   SearchClient,
-  SearchServiceClient,
+  SearchIndexClient,
+  SearchIndexerClient,
   AzureKeyCredential
 } = require("@azure/search-documents");
 
 // To query and manipulate documents
-const indexClient = new SearchClient(
+const searchClient = new SearchClient(
   "<endpoint>",
   "<indexName>",
   new AzureKeyCredential("<apiKey>")
 );
 
-// To manage indexes, datasources, skillsets and more
-const serviceClient = new SearchServiceClient("<endpoint>", new AzureKeyCredential("<apiKey>"));
+// To manage indexes and synonymmaps
+const indexClient = new SearchIndexClient("<endpoint>", new AzureKeyCredential("<apiKey>"));
+
+// To manage indexers, datasources and skillsets
+const indexerClient = new SearchIndexerClient("<endpoint>", new AzureKeyCredential("<apiKey>"));
```

Comment:
no example of using indexerclient?

Reply:
Added one example of creating skillset. Also, I am working on next task to add samples (under samples folder). That will include examples of all the APIs